docs/API.md

```
@@ -0,0 +1,688 @@
+
+---
+
+## Table of Contents
+
+- [Authentication](#authentication)
+  - [Web Session (JSESSIONID)](#web-session-jsessionid)
+  - [App Token Flow](#app-token-flow)
+- [Device State Endpoints](#device-state-endpoints)
+  - [GET /device/list/v2 — Device List v2](#get-devicelistv2--device-list-v2)
+  - [GET /device/list — Device List v1](#get-devicelist--device-list-v1)
+  - [GET /device/deviceInfo/now/{id} — Real-time Info (Unreliable)](#get-devicedevicenowid--real-time-info-unreliable)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)